mmv1/products/containerattached/Cluster.yaml

```diff
@@ -337,3 +337,26 @@ properties:
         values:
           - :DISABLED
           - :PROJECT_SINGLETON_POLICY_ENFORCE
+  - !ruby/object:Api::Type::NestedObject
+    name: proxyConfig
```

Comment:
Can any of these values be changed without recreating the whole cluster? If so, the update tests should probably be changed to alter the values and verify the update works. If not, then some of these fields should probably have `immutable: true` so that they instruct Terraform to recreate the cluster.

Reply:
Good point. We do allow updating in place wothout having to recreate the cluster.
Updated the test to have this scenario.